framework/source/class/qx/bom/AnimationFrame.js

```diff
@@ -76,11 +76,13 @@ qx.Bootstrap.define("qx.bom.AnimationFrame",
      * soon as the given duration is over.
      *
      * @param duration {Number} The duration the sequence should take.
+     *
+     * @ignore(performance.*)
      */
     startSequence : function(duration) {
       this.__canceled = false;
 
-      var start = +(new Date());
+      var start = (window.performance && window.performance.now) ? (performance.now() + qx.bom.AnimationFrame.__start) : Date.now();
```

Comment:
`performance.timer` is intended to be used for performance monitoring, only. I guess the only reason why it is used here (the only location in the complete framework) is that the animation frame callback gets a `DOMHighResTimeStamp` parameter. Not sure if it is usable for something else, though. I'm wondering what for you need a 5µ resolution an a web platform.